konnectivity-client/pkg/client/client.go

```diff
@@ -114,13 +114,13 @@ func (t *grpcTunnel) serve(c clientConn) {
 			t.pendingDialLock.RUnlock()
 
 			if !ok {
-				klog.V(1).Infoln("DialResp not recognized; dropped")
+				klog.V(1).InfoS("DialResp not recognized; dropped", "connectionID", resp.ConnectID, "dialID", resp.Random)
```

Comment:
I dislike that we call it "random" on most of the other log statements but we do mostly call it that. Can we change other locations which use "random" to use "dialID"?

Reply:
Done (let me know if I missed any)